tests/bsd/ns_get_put.c

```diff
@@ -0,0 +1,20 @@
+#include <assert.h>
+#include <arpa/nameser.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdint.h>
+
+int main(int argc, char *argv[]) {
+	uint8_t *buf = malloc(256);
+	uint16_t original16 = 4891;
+	uint32_t original32 = 4861984;
+	memset(buf, 0x0, sizeof(*buf));
+	ns_put16(original16, buf);
+	uint16_t result16 = ns_get16(buf);
+	assert(result16 == original16);
+	memset(buf, 0x0, sizeof(*buf));
+	ns_put32(original32, buf);
+	uint32_t result32 = ns_get32(buf);
+	assert(result32 == original32);
+	return 0;
```

Comment:
It would also be nice that we also test the absolute results, and don't just check if they are internally consistent.

Reply:
I'm kinda lost on what you want here. As far as I can tell this tests the absolute results too no?

Reply:
Hmm yeah I guess.